submissions/dafen173/js-dom/style.css

```diff
@@ -0,0 +1,163 @@
+
+
+.maincontainer {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+
+
+.mainmenubtn {
+    background-color: #696969;
+    color: white;
+    cursor: pointer;
+    padding:20px;
+    
+}
+
+/*
+.mainmenubtn:hover {
+    background-color: green;
+    }
+*/   
+
+.dropdown {
+    /*position: relative;*/
+    /*display: inline-block;*/
+    float: left;
+    width: 20%;
+}
+
+.dropdown-child {
+    background: linear-gradient(to top, #FFFFFF, #696969);
+}
+
+
+
+
+.dropdown-child div {
+    cursor: pointer;
+    color: #000000;
+    padding: 20px 0 20px 20px;
+    text-decoration: none;
+    display: block;
+    
+}
+
+.dropdown-child div:hover {
+    color: #FF8C00;
+    background-color: #696969;
+}
+
+
+
+
+
+
+.content {
+    width: 80%;
+}
+
+h1 {
+    text-align: center;
+}
+
+img {
+    border-radius: 3%;
+    width: 90%;
+}
+
+.container {
+    padding: 20px 20px 40px 20px; 
+}
+
+
+.pic {
+  display: block;
+  margin: 0 auto;
+}
+
+
+.pic2 {
+  display: block;
+  margin: 0 auto;
+}
+
+
+
+
+@media screen and (max-width: 750px) {
+    .dropdown-child {
+            display: none;
+        }
+
+    .open {
+            display: block;
+        }
+
+   
+
+    .dropdown {        
+            width: 100%;
+    }
+}
+
+
+
+
```

Comment:
Remove commented code(use another feature branch for saving these code) and remove empty 
unused lines.